pkg/testhelper/testhelper.go

```diff
@@ -111,20 +109,8 @@ func CompareWithFixture(t *testing.T, output interface{}, opts ...Option) {
 		t.Fatalf("failed to read testdata file: %v", err)
 	}
 
-	diff := difflib.UnifiedDiff{
-		A:        difflib.SplitLines(string(expected)),
-		B:        difflib.SplitLines(string(serializedOutput)),
-		FromFile: golden,
-		ToFile:   "Current",
-		Context:  3,
-	}
-	diffStr, err := difflib.GetUnifiedDiffString(diff)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	if diffStr != "" {
-		t.Errorf("got diff between expected and actual result: \n%s\n\nIf this is expected, re-run the test with `UPDATE=true go test ./...` to update the fixtures.", diffStr)
+	if diff := cmp.Diff(expected, serializedOutput); diff != "" {
```

Comment:
These need to be converted to strings first

Reply:
Why? It seemed to work fine without that.